### PR TITLE
[camera][android]:catch_IllegalArgumentException exception.

### PR DESCRIPTION
**Pull request by arc-yong**
Catch IllegalArgumentException exception for CameraCaptureSession setRepeatingRequest call.

Fixes https://github.com/flutter/flutter/issues/117945

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

File: packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java [hunk 1]
```diff
@@ -529,6 +529,8 @@ void refreshPreviewCaptureSession(
       onErrorCallback.onError("cameraAccess", "Camera is closed: " + e.getMessage());
     } catch (CameraAccessException e) {
       onErrorCallback.onError("cameraAccess", e.getMessage());
+    } catch (IllegalArgumentException e) {
+      onErrorCallback.onError("cameraAccess", e.getMessage());
```
Comment (camsim99) [inline packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java:533]: Based on the [documentation](https://developer.android.com/reference/android/hardware/camera2/CameraCaptureSession#setRepeatingRequest(android.hardware.camera2.CaptureRequest,%20android.hardware.camera2.CameraCaptureSession.CaptureCallback,%20android.os.Handler)) for `setRepeatingRequest`, it looks like this exception will be thrown if there's an issue with `Surface` configuration, which would suggest an internal implementation (`Camera.java`) issue, so I'm not sure we would consider this a camera error, but rather an issue we'd need to investigate.

Are you encountering this issue https://github.com/flutter/flutter/issues/117945?

**Review by camsim99 — COMMENTED** [review]
Hi, thank you for the contribution! I left a comment to get more context about the intention behind this fix.

File: packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java (hunk 1 above)
Comment (arc-yong) [reply packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java:533]: Yes. We are encountering this issue https://github.com/flutter/flutter/issues/117945.

**Comment by stuartmorgan** [comment]
Thanks for the submission!

In the future, please do not delete the checklist that is in the PR template; it is there for a reason. This PR is missing required elements described in the checklist (I’ve restored it to the PR description), which need to be addressed before it moves forward with review.

I am marking the PR as a Draft. Please review the checklist, updating the PR as appropriate, and when that’s complete please feel free to mark the PR as ready for review.

File: packages/file_selector/file_selector_android/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 0.5.0+5
```
Comment (stuartmorgan) [inline packages/file_selector/file_selector_android/CHANGELOG.md:1]: Why is this PR changing the version of a lot of unrelated packages?

File: packages/camera/camera_android/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 0.10.8+14
+
+* Fixes issue https://github.com/flutter/flutter/issues/117945z.
```
Comment (stuartmorgan) [inline packages/camera/camera_android/CHANGELOG.md:3]: This is a broken link. Also, the changelog should describe fixes, not just link to an issue.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
> - [x] I added new tests to check the change I am making, or this PR is [test-exempt](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests).

Where is this new test?

**Comment by arc-yong** [comment]
Replace this with a new PR https://github.com/flutter/packages/pull/5234

**Pull request closed without merging**